src/main/java/org/opensearch/security/securityconf/ConfigModelV6.java

```diff
@@ -791,6 +800,11 @@ public String getDlsQuery(User user) {
             return replaceProperties(dlsQuery, user);
         }
 
+        @Override
+        public Set<String> concreteIndexNames(User user, IndexNameExpressionResolver resolver, ClusterService cs) {
```

Comment:
Nit: Same nit as below. Can we swap the inherited method to a different name if this is something added in this PR? I don't understand the concrete terminology here and I think we may be able to find a more intuitive convention.